src/_imaging.c

```diff
@@ -121,7 +122,7 @@ typedef struct {
     ImagingAccess access;
 } ImagingObject;
```

Comment:
Is there a way to directly convert between structs like this and an HPy type, without having to go through a PyObject?

Reply:
`HPy_AsStruct`. I think the best knowledge source for now are the tests in the HPy repository. Maybe this is something we can improve in the docs..